### PR TITLE
Update change-log.md

### PR DESCRIPTION
**Pull request by MartaD**
product update 31/10/2022

### CODE REVIEW

**Review by chrishumanitec — APPROVED** [review]
LGTM

File: docs/change-log.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update October 31st, 2022
+
+- **New:** Introduced a new API and updated UI which allow for image management at scale. Users can now identify, filter, archive and update images and their versions with ease. More detailed information on new capabilities can be found in the [feature announcement article](https://humanitec.com/blog/container-image-management-made-hassle-free-feature-announcement) on Humanitec's blog.
+![advanced_image_management](_assets/images/2022-10-31_Advanced_Image_Management.png)
+
+- **Fixed:** Provisioning a resource with an invalid [placeholder](https://docs.humanitec.com/reference/concepts/app-config/placeholders) now returns an expected error code with more descriptive error message.
```
Comment (sujaya-sys) [inline docs/change-log.md:10]: "now returns an expected error code with more descriptive error message."

I'd reword this to:

"now returns an error code with a descriptive error message."

Also "improved" might make more sense here as a category (compared to "fixed")

File: docs/change-log.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update October 31st, 2022
```
Comment (sujaya-sys) [inline docs/change-log.md:5]: We always publish product updates on Fridays... I know we're a little late this time but I just always set the date to the previous Friday to keep it consistent.

File: docs/change-log.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
...
+
+- **New:** Introduced a new API and updated UI which allow for image management at scale. Users can now identify, filter, archive and update images and their versions with ease. More detailed information on new capabilities can be found in the [feature announcement article](https://humanitec.com/blog/container-image-management-made-hassle-free-feature-announcement) on Humanitec's blog.
+![advanced_image_management](_assets/images/2022-10-31_Advanced_Image_Management.png)
```
Comment (sujaya-sys) [inline docs/change-log.md:8]: We use 3 standard widths for images that we publish: 720, 480, 240

File: docs/change-log.md (hunk 1 above)
Comment (MartaD) [reply docs/change-log.md:10]: the issue was that it returned an incorrect error code, 500 instead of 400, and that was fixed. Happy to change to your suggestion if it's not that important.
